### DISCUSSION TITLE
How do I check the rolling update status?

### DISCUSSION THREAD

**Discussion by jrivers96** [strimzi/strimzi-kafka-operator#4373] [General]
How do I check the kafka and zookeeper roll status? 

```
kubectl  annotate statefulset kafka-cluster-zookeeper strimzi.io/manual-rolling-update=true
kubectl annotate statefulset kafka-cluster-kafka strimzi.io/manual-rolling-update=true
```
Something like this?

`kubectl rollout status sts/kafka-cluster-kafka`

**Comment by scholzj** [comment]
TBH, I do not think there is any way how you can check it.

**Reply by jrivers96** [reply]
We use this now:
`kubectl  wait kafka/kafka-cluster --for=condition=Ready --timeout=2h`

The use case is for upgrades that occur at night. We want to be alerted about the state of the upgrade.

**Reply by scholzj** [reply]
But this should complete immediately if the cluster is already in the Ready state, or?
